docs/source/admin/log-messages.md

```diff
@@ -1,33 +1,30 @@
 # Interpreting common log messages
 
 When debugging errors and outages, looking at the logs emitted by
-JupyterHub is very helpful. This document tries to document some common
-log messages, and what they mean.
+JupyterHub is very helpful. This document intends to describe some common
+log messages, and what necessary actions to take.
 
-## Failing suspected API request to not-running server
+## Failing suspected API request to run-down server
```

Comment:
Let's keep the same naming here, to match the one in the actual log message (you can find it in the example below). This will help with making this section more visible for people when searching after a key phrase in their logs.

```suggestion
## Failing suspected API request to not-running server
```